### ISSUE TITLE
Add bar.*.separator_color option

### ISSUE THREAD

**Issue by squigglezworth** [weechat/weechat#975]
It'd be nice to be able to set the separator color per bar

**Comment by flashcode** [comment]
You mean `weechat.bar.xxx.color_delim` or something else?

**Comment by squigglezworth** [comment]
No, I mean a `weechat.bar.XXX.separator_color` option, to color the separator line between bar and other bars/windows

**Comment by squigglezworth** [comment]
This could also be extended to allow `weechat.look.window_separator_(horizontal|vertical)`have different colors as well.
